forge/src/main/java/gg/moonflower/pollen/core/forge/PollenCommonForgeEvents.java

```diff
@@ -171,6 +178,19 @@ public static void onEvent(net.minecraftforge.event.world.ExplosionEvent.Detonat
         ExplosionEvents.DETONATE.invoker().detonate(event.getWorld(), event.getExplosion(), event.getAffectedEntities());
     }
 
+    @SubscribeEvent
+    public static void onEvent(net.minecraftforge.event.world.SaplingGrowTreeEvent event) {
+        InteractionResult result = TreeGrowingEvent.EVENT.invoker().interaction(event.getWorld(), event.getRand(), event.getPos());
+        switch (result) {
```

Comment:
This switch statement needs to have breaks or else all branches will fall back to `event.setResult(Event.Result.DENY);`

Reply:
This issue should hopefully be fixed in the latest commit, I also removed the duplicate case statement